third_party/checker_framework_annotations/LICENSE

```diff
@@ -0,0 +1,35 @@
+A few parts of the Checker Framework have more permissive licenses.
```

Comment:
It seems strange that these just appear from nowhere.

Reply:
This was deleted from https://github.com/bazelbuild/bazel/commit/205ba06a677fd1db428ca3cd154f4e585e8e45d4#diff-37771c2b9a79a4881cd5b681f7ae91c01510e9add7d62c708c86b93035d10ecf, which probably shouldn't happen?  I'm just trying to add it back.

Reply:
But there is no .jar to go along with it.  That is what seems strange. Just a dangling, unreferenced license.  Shouldn't that come from the maven download?

Reply:
The jars are now fetched by rules_jvm_external, but I'm not sure if rules_jvm_external has a way to download license file ? /cc @jin 

@aiuto Note that this license isn't exactly dangling, it'll be globbed into //third_party:srcs and be processed at https://github.com/bazelbuild/bazel/blob/ab71a1002c9c53a8061336e40f91204a2a32c38e/src/main/java/com/google/devtools/build/lib/runtime/commands/license/BUILD#L15

Reply:
Oh. that is subtle. I understand now. We'll have to do some figuring out how to get the right license linkage in the future.